ci/README.md

```diff
@@ -0,0 +1,129 @@
+# CI tools
+
+Continuous Integration tools used by the `rust-bitcoin` org.
+
+#### Table Of Contents
+
+- [Repository](#repository)
+  * [Lock files](#lock-files)
+  * [Crates](#crates)
+  * [Pinning](#dependency-pinning)
+- [Workflows](#workflows)
+- [get_matrix.sh](#get_matrix.sh)
+- [test_vars.sh](#test_vars.sh)
+- [run_task.sh](#run_task.sh)
+  * [Environment variables](#environment-variables)
+  * [Integration tests](#integration-tests)
+  * [Crate specific tests](#crate-specific-tests)
+  * [Fuzzing](#fuzzing)
+
+## Repository
+
+The `run_task.sh` script expects a few things to be present in all repositories.
+
+- lock files: `Cargo-recent.lock` and `Cargo-minimal.lock`
+- crate declarations: `contrib/crates.sh`
+- pinning: `contrib/pin.sh`
+
+## Lock files
+
+All repositories MUST include a minimal and recent lock file:
+
+- `Cargo-recent.lock`: A manifest with some recent versions numbers that pass CI.
+- `Cargo-minimal.lock`: A manifest with some minimal version numbers that pass CI.
+
+The `run_taks.sh` script invokes copies each to `Cargo.toml` and uses `cargo --locked`.
```

Comment:
```suggestion
The `run_tasks.sh` script will overwrite each of these `.lock` files to `Cargo.toml` and uses `cargo --locked`.
```

That was ambiguous. I tried to make more direct and clear.

Reply:
Nice, this made me find that this text is all wrong. lock files are expected to be handled in the workflow not in the `run_task.sh` script.